ssg/build_remediations.py

```diff
@@ -488,7 +488,7 @@ def update_when_from_rule(self, to_update):
             additional_when.append(" or ".join(rule_specific_conditionals))
 
         to_update.setdefault("when", "")
-        new_when = ssg.yaml.update_yaml_list_or_string(to_update["when"], additional_when)
+        new_when = ssg.yaml.update_yaml_list_or_string(to_update["when"], additional_when, prepend=True)
```

Comment:
@ggbecker can you fix this PEP 8 issue?

> Line 491:100: E501 line too long (104 > 99 characters)

```suggestion
        new_when = ssg.yaml.update_yaml_list_or_string(to_update["when"], additional_when,
                                                       prepend=True)
```